docusaurus.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   url: 'https://Seneca-ICTOER.github.io/',
   baseUrl: '/OERTemplate/',
   trailingSlash: false,
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'ignore',
```

Comment:
Why not `'warn'` so you get some info, esp. locally, of problems?

Reply:
good point.  I'll fix that... this broke something and I am not sure what exactly yet.